test/parallel_api/numeric/numeric.ops/transform_scan.pass.cpp

```diff
@@ -232,15 +232,17 @@ test_matrix(UnaryOp unary_op, Out init, BinaryOp binary_op, Out trash)
         invoke_on_all_policies<7>()(test_transform_inclusive_scan_init<In>(), in.begin(), in.end(), out.begin(),
                                     out.end(), expected.begin(), expected.end(), in.size(), unary_op, init, binary_op,
                                     trash);
-        invoke_on_all_policies<8>()(test_transform_inclusive_scan<In>(), in.begin(), in.end(), out.begin(), out.end(),
-                                     expected.begin(), expected.end(), in.size(), unary_op, init, binary_op, trash);
-        invoke_on_all_policies<9>()(test_transform_inclusive_scan_init<In>(), in.cbegin(), in.cend(), out.begin(),
+        invoke_on_all_policies<8>()(test_transform_inclusive_scan_init<In>(), in.cbegin(), in.cend(), out.begin(),
                                     out.end(), expected.begin(), expected.end(), in.size(), unary_op, init, binary_op,
                                     trash);
+#if !TEST_GCC10_TRANSFORM_INCLUSIVE_SCAN_BROKEN
```

Comment:
Could you please describe, what is wrong with `transform_scan` with gcc10?

Reply:
This is an issue for GCC10 vectorizer: https://godbolt.org/z/3r3fT4adG
It's not reproduced with GCC11.1: https://godbolt.org/z/GjK8b3Gxv